app/views/awarded.html

```diff
@@ -1,32 +1,23 @@
+{% extends "layout.html" %} {% block pageTitle %} DfE Maths and Physics Service
+Prototype {% endblock %} {% block beforeContent %} {{ govukPhaseBanner({ tag: {
+text: "beta" }, html: 'This is a new service – your
+<a class="govuk-link" href="#">feedback</a> will help us to improve it.' }) }}
```

Comment:
It's a shame to remove the line breaks here, makes it a lot harder to parse in my head